doc/source/dev/api-dev/array_api.rst

```diff
@@ -300,9 +301,10 @@ for compiled code::
   # array-api-strict and CuPy will always be skipped, for the given reasons.
   # All libraries using a non-CPU device will also be skipped, apart from
   # JAX, for which delegation is implemented (hence non-CPU execution is supported).
-  @pytest.mark.skip_xp_backends('array_api_strict', 'cupy',
-                                reasons=['skip reason 1',
-                                         'skip reason 2',],
+  @pytest.mark.skip_xp_backends('array_api_strict', reason='skip reason 1',
+                                cpu_only=True,
+                                exceptions=['jax.numpy'],)
+  @pytest.mark.skip_xp_backends('cupy', reason='skip reason 2',
                                 cpu_only=True,
                                 exceptions=['jax.numpy'],)
```

Comment:
could you split the `cpu_only=True, exceptions=...` out into a separate decorator call?

Reply:
OK, done.